core/camel-support/src/main/java/org/apache/camel/support/ExtendedExchangeExtension.java

```diff
@@ -30,6 +30,7 @@
 
 public class ExtendedExchangeExtension implements ExchangeExtension {
     private final AbstractExchange exchange;
+    private boolean failureHandled;
```

Comment:
The other flags are in AbstractExchange, so maybe we should move it to where they are.

Reply:
Noted, thanks! I'll take a look at that.